.gitmodules

```diff
@@ -10,3 +10,6 @@
 	path = docs/boilerplate
 	url = https://github.com/aws-quickstart/quickstart-documentation-base-common.git
 	branch = main
+[submodule "quickstart-aws-acm-certificate"]
+	path = quickstart-aws-acm-certificate
+	url = https://github.com/aws-quickstart/quickstart-aws-acm-certificate
```

Comment:
Can we track the "main" branch for this submodule please? Add a `branch = main` line similar to `docs/boilerplate`